osfoffline/settings/defaults.py

```diff
@@ -43,3 +43,9 @@
 
 PROJECT_LOG_DIR = user_log_dir(appname=PROJECT_NAME, appauthor=PROJECT_AUTHOR)
 PROJECT_LOG_FILE = os.path.join(PROJECT_LOG_DIR, 'osfoffline.log')
+
+#updater
+REPO = 'CenterForOpenScience/OSF-Offline'
+VERSION = '0.1.0'
```

Comment:
We've been using the `0.1.0` version for quite some time, even after major rewrites. (this was the version number of the November 18 QA build, for example)

As part of this PR, I suggest changing `VERSION` to `0.2.0` to avoid confusion. Future QA builds can increment from there.